test/framework/events.ts

```diff
@@ -21,9 +22,12 @@ export function event(name: string, receipt: ContractReceipt): Event {
  */
 export function events(name: string, receipt: ContractReceipt): Event[] {
     const availableEvents = receiptEvents(receipt)
+    log.info('looking for event name: ', name)
```

Comment:
issue: the logging isn't there for use during tests. It's mean solely for scripts.

Logging during tests isn't usually helpful as console logging rarely has things synced correctly and file lacks the context of execution.

Reply:
think this was stale somehow! didnt mean to push log.info statements!!